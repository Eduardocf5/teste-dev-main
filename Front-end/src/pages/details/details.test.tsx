import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from "react-router-dom";
import Details from '.';

test('render test', () => {
    const component = renderer.create(<BrowserRouter>
        <Details />
    </BrowserRouter>).toJSON();

    expect(component).toMatchSnapshot();
  });