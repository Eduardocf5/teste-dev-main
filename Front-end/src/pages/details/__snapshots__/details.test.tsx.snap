// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render test 1`] = `
<div
  className="flex justify-center"
>
  <div
    className="bg-white p-4 rounded-md"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="back-arrow"
        className="w-10 cursor-pointer"
        src="arrow-back.svg"
      />
    </a>
    <div
      className="flex items-center flex-wrap gap-9 mt-5"
    >
      <img
        alt="icond"
        className="rounded-full max-w-44"
      />
      <div>
        <div
          className="text-4xl"
        />
        <div
          className="flex items-center gap-1"
        >
          <div
            className="rounded-full undefined w-3 h-3"
          />
          <div>
             - 
          </div>
        </div>
      </div>
    </div>
    <div
      className="mt-4"
    >
      <span
        className="font-semibold"
      >
        Nome de origem: 
      </span>
      <span />
    </div>
    <div
      className="mt-2"
    >
      <span
        className="font-semibold"
      >
        Genero: 
      </span>
      <span />
    </div>
    <div
      className="mt-2"
    >
      <span
        className="font-semibold"
      >
        Localização: 
      </span>
      <span />
    </div>
  </div>
</div>
`;
